feature-examples/python/create_apikey.py

```diff
@@ -0,0 +1,131 @@
+"""Example of using the Vectara REST API to create an API Key."""
+
+import argparse
+import logging
+import sys
+from typing import Any
+
+import requests
+
+import vectara_util
+
+
+def create_apikey(
+    customer_id: int,
+    corpus_id: int,
+    auth_url: str,
+    app_client_id: str,
+    app_client_secret: str,
+) -> tuple[Any, bool]:
+    """Creates an API key.
+
+    Args:
+        customer_id: Unique customer ID in vectara platform.
+        corpus_id: Corpus ID to which API key will be created.
+        auth_url: The cognito auth url for this customer.
+        app_client_id: The id of the app client with enough rights.
+        app_client_secret: The app client secret.
+
+    Returns:
+        (apiKey, True) in case of success and returns (error, False) in case of failure.
+    """
+    jwt_token = vectara_util.get_jwt_token(auth_url, app_client_id, app_client_secret)
+    if not jwt_token:
+        logging.error("Failed to get JWT token.")
+        return "Failed to get JWT token.", False
+
+    post_headers = {
+        "customer-id": f"{customer_id}",
+        "Authorization": f"Bearer {jwt_token}",
+    }
+
+    # A request can contain multiple api keys. We are creating only one.
+    request = {
+        "apiKeyData": [
+            {
+                "description": "API Key for testing.",
+                "apiKeyType": 2,  # 1 - Query, 2 - Query & Indexing
+                "corpusId": [corpus_id],  # One key can be used for multiple corpora.
+            }
+        ]
+    }
+
+    response = requests.post(
+        "https://api.vectara.io/v1/create-api-key",
+        json=request,
+        verify=True,
+        headers=post_headers,
+        timeout=50,
+    )
+
+    if response.status_code != 200:
+        logging.error(
+            "CreateApiKey failed with code %d, reason %s, text %s",
+            response.status_code,
+            response.reason,
+            response.text,
+        )
+        return response, False
+
+    message = response.json()
+    if message["response"]:
+        if len(message["response"]) != 1:
+            logging.error("CreateApiKey failed with response %s", message["response"])
+            return message["response"], False
+        status = message["response"][0]["status"]
```

Comment:
nit: swap lines 75 and 76 so blank line comes after the `len != 1` block and `status` assignment is grouped with the `if status` block.